test/backend_graph_comparison.in.cpp

```diff
@@ -0,0 +1,278 @@
+//*****************************************************************************
+// Copyright 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <cstdlib>
+#include <string>
+#include "gtest/gtest.h"
+
+#include "ngraph/log.hpp"
+#include "ngraph/ngraph.hpp"
+#include "ngraph/runtime/host_tensor.hpp"
+#include "ngraph/serializer.hpp"
+#include "util/all_close.hpp"
+#include "util/all_close_f.hpp"
+#include "util/test_control.hpp"
+
+using namespace ngraph;
+
```

Comment:
It might be better to use `using namespace std;` in this file as it done in all `*.cpp` files in sources.

Reply:
Done.